### ISSUE TITLE
Lora artefacts

### ISSUE THREAD

**Issue by AlexandreLrz** [bghira/SimpleTuner#1754]
I'm not sure if this problem is coming from my (maybe wrong ?) config implementation or something else, but I have this weird rectangle coming in at step 750 in my lora finetuning (in 1 picture over 13 in my validation samples).

<img width="1437" height="379" alt="Image" src="https://github.com/user-attachments/assets/ea59f519-d6c2-4ff8-aed0-9c19dde2df3a" />

Here is my main configuration :
```
{    
    "data_backend_config": "config/multidatabackend.json",
    "checkpointing_steps": 250,
    "checkpoints_total_limit": 40,
    "disable_benchmark": false,
    
    "learning_rate": 6e-5,
    "lr_warmup_steps": 100,
    "lr_scheduler": "constant_with_warmup",
    "flow_schedule_shift": 3,
    "gradient_checkpointing": true,
    "max_grad_norm": 0.05,
    
    "use_ema": false,
    
    "hub_model_id": "yts",
    "model_family": "hidream",
    "model_flavour": "full",
    "mixed_precision": "bf16",
    "base_model_precision": "int8-quanto",
    "model_type": "lora",
    "lora_alpha": 128,
    "lora_rank": 128,
    "lora_type": "standard",

    "quantize_via": "cpu",
    "quantize_activations": false,
    
    "report_to": "wandb",
    "tracker_project_name": "hidream-training_24",
    "tracker_run_name": "training-run_1",

    "push_checkpoints_to_hub": false,
    "push_to_hub": false,

    "train_batch_size": 1,
    "resume_from_checkpoint": "latest",
    "seed": 42,
    "optimizer": "adamw_bf16",
    "max_train_steps": 30000,
    "num_train_epochs": 0,
    "ignore_final_epochs": true,
    "caption_dropout_probability": 0.15,
    "gradient_accumulation_steps": 2,

    "validation_guidance": 5.0,
    "validation_num_inference_steps": 30,
    "validation_resolution": "640x360",
    "validation_seed": 42,
    "validation_steps": 250,
    "validation_torch_compile": false,
    "validation_prompt_library": true,
    "user_prompt_library": "config/user_prompt_library.json",

    "output_dir": "output/",

    "minimum_image_size": 360,
    "maximum_image_size": 2000,
    "target_downsample_size": 2000,
    "resolution_type": "pixel_area",
    "resolution": 512,
    "override_dataset_config": true
}
```

And multidatabackend :

```
[
  {
    "id": "test",
    "type": "local",
    "instance_data_dir": "/workspace/dataset",
    "crop": false,
    "crop_style": "center",
    "crop_aspect": "preserve",
    "minimum_image_size": 360,
    "maximum_image_size": 2000,
    "target_downsample_size": 2000,
    "resolution": 512,
    "resolution_type": "pixel_area",
    "prepend_instance_prompt": false,
    "instance_prompt": "",
    "only_instance_prompt": false,
    "caption_strategy": "filename",
    "cache_dir_vae": "cache/vae/test",
    "vae_cache_clear_each_epoch": false,
    "probability": 1.0,
    "repeats": 1,
    "skip_file_discovery": "",
    "preserve_data_backend_cache": true,
    "text_embeds": "text-embed-cache"
  },
  {
    "id": "text-embed-cache",
    "dataset_type": "text_embeds",
    "default": true,
    "type": "local",
    "cache_dir": "cache/text/test",
    "write_batch_size": 128
  }
]
```

All my dataset pictures have the same size (1280x720) and none of them have artefacts like this.

Thank you for your work by the way it's amazing!

**Comment by bghira** [comment]
what version/commit is this with?

**Comment by AlexandreLrz** [comment]
> what version/commit is this with?

latest

**Comment by bghira** [comment]
```bash
mkdir images/
env SIMPLETUNER_DEBUG_IMAGE_PREP=true ./train.sh
```

if you run this with an empty VAE cache directory, you'll see a bunch of images saved into the images/ dir of each stage of their cropping/preprocessing. can you check this to see if they have the white section in these debug outputs? sorry for the request, i know it will take some time. i appreciate this assistance to identify any cause of the problem.

**Comment by AlexandreLrz** [comment]
> mkdir images/
> env SIMPLETUNER_DEBUG_IMAGE_PREP=true ./train.sh
> if you run this with an empty VAE cache directory, you'll see a bunch of images saved into the images/ dir of each stage of their cropping/preprocessing. can you check this to see if they have the white section in these debug outputs? sorry for the request, i know it will take some time. i appreciate this assistance to identify any cause of the problem.

I just had a try today and the only difference between input and output pictures is the size (1280x720 for input and 960x512 for final output).

**Comment by bghira** [comment]
does this same output issue persist across seeds? it might just be seed-related 🤔 that's some pretty bad luck though, I still do want to keep looking for the reason, but i think this might just be how the model is working as well.

**Comment by AlexandreLrz** [comment]
> does this same output issue persist across seeds? it might just be seed-related 🤔 that's some pretty bad luck though, I still do want to keep looking for the reason, but i think this might just be how the model is working as well.

I'm currently investigating but there's definitely something wrong with my implementation, will open another issue for that.
